nobrainer/brainsiam_train.py

```diff
@@ -0,0 +1,47 @@
+"""
+Extending the SimSiam network architecture to brain volumes
+author: Dhritiman Das
```

Comment:
not sure what this file is doing?

Reply:
this is to instantiate the simsiam architecture to generate the representations from the encoder and predictor (which are instantiated separately in brainsiam.py)